### DISCUSSION TITLE
群辉及阿里ECS部署成功。

### DISCUSSION THREAD

**Discussion by Grouppppp** [oddfar/campus-imaotai#315] [分享部署教程]
群晖DSM 7.2.1-69057 Update 3，Container Manager 20.10.23，使用 1.0.11 docker底包及最新代码自行打包 jar，部署成功。
阿里云ecs Debian12安装docker，使用同样的 1.0.11 docker底包及最新代码自行打包 jar，部署成功。

**Comment by ljmind** [comment]
求大佬给个主要步内容

**Comment by Rolanddaan** [comment]
有没有大佬给点详细步骤啊，现在我这部署完成，报错接口404
